client/src/services/content.ts

```diff
@@ -157,15 +157,22 @@ Dit kan worden gebruikt voor:
     },
     footer: {
         column1: `# Contact
-Hebt u een vraag en kunt u het antwoord niet vinden op deze website? Neem dan contact met ons op.`,
+Hebt u een vraag en kunt u het antwoord niet vinden op deze website? Neem dan contact met ons op.
+- [Contactformulier](/)
+- **Bel het telefoonnummer [14 020](/)**
```

Comment:
van 14020 zou ik `(/)` veranderen in `(tel:14020)`, dat is de link naar een telefoonnummer op mobiel

Reply:
Goed voorstel. Ik heb eigenlijk al die links 'naar buiten' voor nu niet actief gemaakt. Maar akkoord met je voorstel.